js/main.js

```diff
@@ -374,8 +374,8 @@ function renderRow(info, section) {
     '<div class="route animated ' + options.start_animation +
     '" style="background-color: #' + info.Route.Color + '; color: #' + info.Route.TextColor + '">' +
       '<div class="row">' +
-        '<div class="route_short_name ' + short_proportions + '">' +
-          info.Route.ShortName +
+        '<div class="route_abbreviation ' + short_proportions + '">' +
```

Comment:
I updated this class name, but couldn't find it used anywhere in the CSS. Perhaps we could just write

```js
'<div class="' + short_proportions + '">' + //...
```
?

Reply:
Yeah, the CSS got removed in #51 I think it's OK to leave it, though.